src/gui/editors/PianoRoll.cpp

```diff
@@ -2454,11 +2454,11 @@ void PianoRoll::mouseMoveEvent( QMouseEvent * me )
 								( (float)( noteEditBottom() - noteEditTop() ) ) *
 								( MaxVolume - MinVolume ),
 											MaxVolume );
-				pan = qBound<int>( PanningLeft,
-								PanningLeft +
+				pan = qBound<panning_t>( PanningLeft,
```

Comment:
Also about this one?

Reply:
Here the compiler was complaining that mixed types were given to `qBound` and therefore I changed it to the type that it is actually working with.

Reply:
ok